events/EiffelActivityStartedEvent/simple.json

```diff
@@ -0,0 +1,22 @@
+{
+  "meta": {
+    "type": "EiffelActivityStartedEvent",
+    "version": "1.0",
+    "time": 1234567890,
+    "domainId": "example.domain",
+    "id": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee"
+  },
+  "data": {
+    "executionUri": "https://my.jenkins.host/myJob/43",
+    "liveLogs": [
+      {
+       "name": "logfile",
```

Comment:
The definition of name is "The name of the log file." which I think is wrong. "name" should not be the name of the "log file" but of the "log". Suggest change to "Build console" and that we fix the definition.